tests/unit/config/config_test.py

```diff
@@ -1322,6 +1322,36 @@ def test_load_bind_mount_relative_path(self):
         assert mount.type == 'bind'
         assert mount.source == expected_source
 
+    @mock.patch.dict(os.environ)
+    def test_load_bind_mount_relative_path_with_tilda(self):
+        if IS_WINDOWS_PLATFORM:
```

Comment:
I think we can do without that - assume that Python expands `~` correctly on all platforms, and change the test to verify that `mount.source` doesn't start with `~` and has `/web` as the last element of its path.

Reply:
I updated the test as you suggest, but the result is the same.